### PR TITLE
Feature: navbar proposal w/webpack

### PR DESCRIPTION
**Pull request by bspeare**

### CODE REVIEW

File: src/components/proposals/brand/navbar/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+  h1 >a {
+    display: flex;
+    align-items: center;
+    margin: 0 auto 0 0;
+    text-align: left;
+    font-size: 1.5rem;
+    font-weight: 300;
+    line-height: 1;
+    font-family: "Roboto", sans-serif;
+    letter-spacing: -0.02em;
```
Comment (bspeare) [inline src/components/proposals/brand/navbar/navbar.scss:42]: Do you mind if we remove the negative `letter-spacing`?  We aren't doing that anywhere else with the brand headers.

File: src/components/proposals/brand/navbar/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+    padding: 0 10px;
+    align-items: center;
+   // margin: 0 auto;
+    text-align: center;
+    height: 50px;
+    width:100%;
+    font-size: 1.1375rem;
+
+    background-color: white;
+    border-bottom: 1px solid rgba(0,0,0,0.2);
```
Comment (bspeare) [inline src/components/proposals/brand/navbar/navbar.scss:154]: Add a `last-child` rule to remove final border on final link and nested links?

File: src/components/proposals/brand/navbar/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,466 @@
...
+    -webkit-box-shadow: 0 5px 10px -2px rgba(0, 0, 0, 0.26);
+  }
+
+  .site-links.responsive a {
+    color: $secondary;
+    text-decoration: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0 10px;
```
Comment (bspeare) [inline src/components/proposals/brand/navbar/navbar.scss:145]: Add a little more padding here to match the padding on our Superfish menus?  `padding: .45rem 1rem`

File: src/components/proposals/brand/navbar/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,466 @@
...
+    background-color: white;
+    border-bottom: 1px solid rgba(0,0,0,0.2);
+    border-top: 1px solid rgba(255,255,255,0.05);
+  }
+
+  .dropdown .dropdown-content{
+
+    a{
+      color: #666;
+      font-weight: 400;
```
Comment (bspeare) [inline src/components/proposals/brand/navbar/navbar.scss:162]: Change font-weight to `font-weight: 600;` to match Superfish styles?  `400` is fine for child links.

File: src/components/proposals/brand/navbar/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,466 @@
...
+    }
+
+    a >span>i:last-child{
+      margin-left:5px;
+    }
+
+    a>span>i:first-child{
+      margin-right:5px;
+    }
+
```
Comment (bspeare) [inline src/components/proposals/brand/navbar/navbar.scss:298]: Add a rule to adjust icon size and color?  Something smaller and a different color like: 
```    
color: #fff;
font-size: .7rem;
line-height: 1.4;
```

File: src/assets/scss/uids.scss [hunk 6]
```diff
@@ -8,46 +8,51 @@
...
+'../../components/menus/top/top',
+'../../components/menus/user-menu/user-menu',
+'../../components/menus/vertical/vertical',
+'../../components/pagination/pagination',
+'../../components/search-toggle/search-toggle',
+'../../components/stat/stat.scss',
+'../../components/tables/tables.scss',
+'../../components/tabs/tabs.scss',
+'../../components/video/vertical/vertical',
+'../../components/video/video-utilities/video-utilities';
```
Comment (pyrello) [inline src/assets/scss/uids.scss:56]: Why is this change included?

File: src/components/proposals/brand/navbar/navbar.twig [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+                    <path class="st0" d="M93.8 90.2h-29c-10.5 0-17.4-6.9-17.4-18.2V18.2C47.4 7 54.4 0 64.8 0h29c10.5 0 17.4 7 17.4 18.2V72C111.2 83.2 104.2 90.2 93.8 90.2zM85.6 71.2V18.8H73v52.4H85.6z" />
+                    <path class="st0" d="M122.6 18.8h-6.4V0h38v18.9H147l6.5 43.4L167 0h19.2l14.4 62.3 5.2-43.4h-6.6V0h37.5v18.9h-6.2l-11.3 71.4h-30.6l-11.8-53.2 -12.1 53.1h-29.4L122.6 18.8z" />
+                    <path class="st0" d="M230.1 71.2h6.9L250.7 0h41l13.5 71.2h6.4v19H281l-2.9-22h-15.2l-2.7 22h-30L230.1 71.2 230.1 71.2zM276.5 51.7l-5.8-36.4 -6 36.4H276.5z" />
+                    <image src="../../images/uiowa-primary.png">
+                        <title id="logo-header">University of Iowa</title>
+                    </image>
+                </svg>
+            </a>
+        </div>
+
```
Comment (pyrello) [inline src/components/proposals/brand/navbar/navbar.twig:22]: This should be replaced with a reference to the logo template so that we are not maintaining this fragment in multiple places.

File: src/components/proposals/brand/navbar/navbar.twig [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+                <span class="icons">
+                  <i class="fas fa-chevron-down "></i>
+                </span>
+              </a>
+
+              <div class="dropdown-content" id="target2">
+                  <a href="#">Link 1</a><span></span>
+                  <a href="#">Link Example 2</a><span></span>
+                  <a href="#">Link 3</a><span></span>
+              </div>
```
Comment (pyrello) [inline src/components/proposals/brand/navbar/navbar.twig:87]: Ideally, this would not have demo content directly in the template. This could instead be set in `navbar.config.yml`

File: webpack/readme.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
+# Webpack UIDS build
```
Comment (pyrello) [inline webpack/readme.md:1]: If we are going to add something like this, I would like to see how we could do it for the entire library. I would like to discuss this idea some more.

**Comment by bspeare** [comment]
Closing this for now.
